code-challenges.test.js

```diff
@@ -15,40 +15,104 @@
 
+
+// Create Jest test using syntax example from syllubus with describe, it, and expect statements.
+describe("shuffled", () => {
+
 var colors1 = ["purple", "blue", "green", "yellow", "pink"]
-// Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
+  // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
 var colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
-// Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+  // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+
+  it("takes an array of strings, removes index 0, and shuffles the rest", () => {
+
+  // set expect statement to equal the array, with index 0 deleted, and the rest shuffled.
+    expect(["blue", "green", "yellow", "pink", "new"]).toEqual(expect.arrayContaining(shuffled(colors1)))
+    expect(["indigo", "periwinkle", "ochre", "aquamarine", "saffron"]).toEqual(expect.arrayContaining(shuffled(colors2)))
 
+  })
+})
 
 
 // b) Create the function that makes the test pass.
 
+// create function called shuffled
+const shuffled = (array) => {
+// use .shift() method to remove index 0
+  array.shift()
+// use .sort() method to shuffle the rest of the array, and return the array.
+return array.sort((a, b) => 0.5 - Math.random())
+}
+// Note: function produces green "passed" message. Function refactored, as needed.
+
 
 
 // --------------------2) Create a function that takes an array of numbers and returns an array of the minimum and maximum numbers in that order.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-var nums1 = [3, 56, 90, -8, 0, 23, 6]
-// Expected output: [-8, 90]
-var nums2 = [109, 5, 9, 67, 8, 24]
-// Expected output: [5, 109]
+// Create Jest test using syntax example from syllubus with describe, it, and expect statements.
+describe("minmax", () => {
+
+  var nums1 = [3, 56, 90, -8, 0, 23, 6]
+  // Expected output: [-8, 90]
+  var nums2 = [109, 5, 9, 67, 8, 24]
+  // Expected output: [5, 109]
+
+    it("takes an array of numbers and returns an array of the minimum and maximum numbers in that order", () => {
 
+      // set expect statement to equal an array with the lowest and highest number of the input array
+      expect(minmax(nums1)).toEqual([-8, 90])
+      expect(minmax(nums2)).toEqual([5, 109])
+
+    })
+  })
 
 
 // b) Create the function that makes the test pass.
 
+// create a function called minmax
+const minmax = (array) => {
+// create an empty return array
+  let newArray = []
+// push the min and max values into the return array
+  newArray.push(Math.min(... array))
+  newArray.push(Math.max(... array))
+// return the return array
+  return newArray
+}
+// Note: function produces green "passed" message. Function refactored, as needed.
+
```

Comment:
Looks great. Just to refine a bit, you have all the logic you need for the min and max. You might shorten it up by just putting them into the array. `return [Math.min(...array), Math.max(...array)]`